### PR TITLE
Remove dApp Staking v2 code

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Removed dApp staking v2 code and some other unused code. Also made adjustments to code which referenced removed code.

Things to do
- remove unused translation (tricky and time consuming one)
- integration tests

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/staking-v3/hooks/useDapps.ts [hunk 1]
```diff
@@ -99,6 +106,72 @@ export function useDapps() {
...
+      eventAggregator.publish(
+        new ExtrinsicStatusMessage({
+          success: true,
+          message: t('stakingV3.registration.success', { name: parameters.dapp.name }),
+        })
+      );
+
+      return true;
+    } catch (e) {
+      const error = e as unknown as AxiosError;
```
Comment (gtg7784) [inline src/staking-v3/hooks/useDapps.ts:160]: lets use `axios.isAxiosError` for error handling

https://github.com/axios/axios?tab=readme-ov-file#typescript

**Review by impelcrypto — APPROVED** [review]
Tested and it works. 
I think you can move the files from `staking-v3` (this can be done in this or the other PR)

**Review by gtg7784 — CHANGES_REQUESTED** [review]

**Comment by impelcrypto** [comment]
@bobo-k2 I've resolved conflict in [this](https://github.com/AstarNetwork/astar-apps/pull/1283/commits/8e31b0a9fa80b50ae1f30e8aceb0fabd5300c2dd) commit.

**Comment by bobo-k2** [comment]
@impelcrypto, @gtg7784 thank you for comments. I resleoved them in https://github.com/AstarNetwork/astar-apps/pull/1283/commits/8e31b0a9fa80b50ae1f30e8aceb0fabd5300c2dd

**Comment by Kahonnohak** [comment]
start checking now and found dApp news is missing 🫣
Screenshot 2024-05-15 at 15 42 42

**Comment by Kahonnohak** [comment]
Could you remove data from dApp news section apart from the latest 5? Keep what we can see and remove all (only promotion ones)
Screenshot 2024-05-15 at 15 45 08

**Comment by bobo-k2** [comment]
> Tested and it works. I think you can move the files from `staking-v3` (this can be done in this or the other PR)

I will move in another PR

**Comment by bobo-k2** [comment]
> Could you remove data from dApp news section apart from the latest 5? Keep what we can see and remove all (only promotion ones) 

@Kahonnohak thank you for comments. I fixed missing ads area. I am not sure if I get comment regarding number of dApps correctly. I changed the code to show 5 promoted and 5 new dApps. Let me know if needs to be different.

Fixed in https://github.com/AstarNetwork/astar-apps/pull/1283/commits/db3a394a52b66fb7d094be1e3c151715fe62176c

**Review by gtg7784 — APPROVED** [review]
LGTM, thanks!

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Pull request merged**